Modules/CTP/src/RawDataReaderCheck.cxx

```diff
@@ -99,6 +99,8 @@ Quality RawDataReaderCheck::check(std::map<std::string, std::shared_ptr<MonitorO
   if (lhcifdata == nullptr) {
     ILOG(Info, Support) << "LHC data not found for timestamp:" << mTimestamp << ENDM;
     return result;
+  } else {
+    ILOG(Info, Support) << "LHC data found for timestamp:" << mTimestamp << ENDM;
```

Comment:
This message will be printed every time the check is called. Is it what you want ? shouldn't it be a debug message ?

Reply:
outdated